app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -228,4 +229,19 @@ class LoginActivity : AppCompatActivity(), ILoginView {
             loginPresenter.requestPassword(email, url, isPersonalServerChecked)
         }
     }
+    private fun backhandle(context: Context) {
+        val lastfragment = supportFragmentManager.findFragmentById(R.id.fragment_container)
+        if (lastfragment == null) {
```

Comment:
This is wrong, this will open ChatActivity unconditionally if I am in LoginActivity

We don't want these enhancements, please focus on relevant features and bug fixes